taskmanager/taskeditor/templates/main/index.html

```diff
@@ -22,12 +21,25 @@ <h2>Your task manager</h2>
               <li><a class="dropdown-item rounded-2" href="{% url 'home' status='completed' %}">Completed tasks</a></li>
               <li><a class="dropdown-item rounded-2" href="{% url 'home' %}">Uncompleted tasks</a></li>
           </ul>
+              {% if groups %}
+               <ul class="dropdown-menu position-static d-grid gap-1 p-2 rounded-3 mx-0 shadow w-220px" data-bs-theme="light">
+                  <li><a class="dropdown-item rounded-2" href="#">Filter groups</a></li>
+                  <li><hr class="dropdown-divider"></li>
+                   {% for elem in groups %}
+                        <li><a class="dropdown-item rounded-2" href="{% url 'home' status=elem.id%}">{{ elem.name}}</a></li>
+                       <button class="btn btn-outline-secondary" type="button">
```

Comment:
Можно вкладывать только ul → li, другие элементы нельзя

```suggestion
                        <li><a class="dropdown-item rounded-2" href="{% url 'home' status=elem.id%}">{{ elem.name}}</a>
                       <a href="{% url 'delete_group' elem.id %}" class="btn btn-outline-secondary" type="button">
```